### PR TITLE
New Template: Cisco IOS BFD neighbors details

### PR DESCRIPTION
**Pull request by mjbear**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT

cisco_ios_show_bfd_neighbors_details, cisco_ios, show bfd neighbors [ipv4|ipv6|client PROTOCOL] details

##### SUMMARY

Support parsing detailed BFD neighbor output. The template can parse IPv4, IPv6, or protocol specific BFD neighbors (see examples below).

Example Commands:
show bfd neighbors details
show bfd neighbors ipv6 details
show bfd neighbors client bgp details

```

```

### CODE REVIEW

**Review by jvanderaa — APPROVED** [review]

File: ntc_templates/templates/cisco_ios_show_bfd_neighbors_details.textfsm [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Start
+  ^\s*IPv\d+\s+Sessions\s*$$
+  ^\s*(?:\S+\s+)?NeighAddr.*$$ -> BFD
+  #
+  # Capture time-stamp if vty line has command time-stamping turned on
+  ^Load\s+for\s+
+  ^Time\s+source\s+is
+  ^\s*$$
+
```
Comment (jvanderaa) [inline ntc_templates/templates/cisco_ios_show_bfd_neighbors_details.textfsm:39]:
```suggestion
  ^. -> Error

```

**Review by jvanderaa — CHANGES_REQUESTED** [review]
Add of the error on the Start and should be all set.

**Review by jvanderaa — APPROVED** [review]

**Comment by mjbear** [comment]
@jvanderaa : should the commit history have been re-written?
(or should have squashed them all into one "pretty" commit message?)

Notice how there a slew of commit messages from me on April 5 and April 11.
https://github.com/networktocode/ntc-templates/commits/master

**Comment by jvanderaa** [comment]
@mjbear I'll check into that with @FragmentedPacket, @jmcgill298 and get that backed down to a single.

**Comment by jvanderaa** [comment]
@mjbear it should be all set now. Thanks @FragmentedPacket

**Comment by mjbear** [comment]
> @mjbear it should be all set now. Thanks @FragmentedPacket

Clean, so clean!
Much appreciated @jvanderaa and @FragmentedPacket

**Pull request merged**